### ISSUE TITLE
Lab 1 

### ISSUE THREAD

**Issue by jasminacosta** [Watts-College/paf-510-template#2]
Hi Professor, for lab #1 question 2 when we interpret b1 in plain English does it mean to say that we write out that every time x goes up by 2 units y is predicted to go up by the value that we get whenever we solve for b1? I am just unsure how to write it out in words. 

Thank you!

**Comment by youngjaewon** [comment]
Hi Jasmin, I don't know why I didn't get a notification for your post. I apologize for the delayed response. Everything you said is correct, but x goes up by "1 unit", not 2 units.

**Comment by jasminacosta** [comment]
No worries! That makes a lot of sense. 

Thank you!
